CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+Guidelines
+==========
+
+Contributions to `earthpy` are welcome. They are likely to
+be accepted more quickly if they follow these guidelines.
+
+At this stage of development, we are developing a set of
+usable wrapper functions that help make working with earth
+systems data easier.
+
+Edits & Updates
+===============
+
+When submitting a change to the repository, please do the following:
+
+- Please first create an issue that covers the item that you'd like to change, update or enhance. Once a discussion has yielded a vote of support for that addition to the package, you are ready to submit a pull request.
+
+When submitting a pull request:
+
+- All existing tests should pass. Please make sure that the test
+  suite passes, both locally and on
+  [Travis CI](https://travis-ci.org/earthlab/earthpy). Status on
+  Travis will be visible on a pull request. If you want to enable
+  Travis CI on your own fork, please read the pandas guidelines link
```

Comment:
@lwasser what's the reference to the "pandas guidelines" link? Is this a holdover from the source file?